mkdocs.yml

```diff
@@ -47,10 +48,11 @@ pages:
         - 'Enable Wi-Fi on Arduino MKR1000': 'os/tutorials/wi-fi_on_arduino.md'
         - 'Write a Test Suite for a Package': 'os/tutorials/unit_test.md'
         - 'Events and Event Queues': 'os/tutorials/event_queue.md'
-        - 'Project Slinky for remote comms':
-            - 'Slinky on sim device': 'os/tutorials/project-slinky.md'
-            - 'Slinky on Nordic nRF52 board': 'os/tutorials/project-target-slinky.md'
-            - 'Slinky on STM32 board': 'os/tutorials/project-stm32-slinky.md'
+        - 'Project Slinky for Remote Comms':
+            - toc: 'os/tutorials/project-slinky.md'
+            - 'Slinky on sim device': 'os/tutorials/project-sim-slinky.md'
+            - 'Slinky on Nordic nRF52': 'os/tutorials/project-nrf52-slinky.md'
+            - 'Slinky on Olimex': 'os/tutorials/project-stm32-slinky.md'
```

Comment:
I will reorder these a little: move up Slinky and Checks stats in the list so people can go down the list in increasing order of complexity/new concepts.